hr_expense_invoice/models/hr_expense.py

```diff
@@ -56,5 +89,5 @@ def _onchange_invoice_id(self):
         """
         if self.invoice_id:
             self.quantity = 1
-            self.unit_amount = self.invoice_id.amount_total
             self.tax_ids = [(5,)]
+            self.unit_amount = self.invoice_id.amount_total
```

Comment:
Why this change or order?

Reply:
It's very important because if assign `unit_amount` before se `tax_ids` to empty, invoice constraint show (`_check_expense_ids`). 
These little change it's necessary to "_Create Vendor Bill_" button in expense work fine.

Reply:
I'm not sure to get you. Are you saying that a constraint raises? Which one? In any case, if so, please put a comment like:
```python
# Assign this amount after removing taxes for avoiding to raise the constraint X
```
and remember always that when something is not obvious, you should put a comment for that.